homeassistant/components/zwave/const.py

```diff
@@ -324,14 +324,30 @@
 DISC_GENRE = "genre"
 DISC_INDEX = "index"
 DISC_INSTANCE = "instance"
-DISC_LABEL = "label"
 DISC_NODE_ID = "node_id"
 DISC_OPTIONAL = "optional"
 DISC_PRIMARY = "primary"
-DISC_READONLY = "readonly"
+DISC_SCHEMAS = "schemas"
 DISC_SPECIFIC_DEVICE_CLASS = "specific_device_class"
 DISC_TYPE = "type"
 DISC_VALUES = "values"
-DISC_WRITEONLY = "writeonly"
 
-METER_RESET_INDEX = 33
+INDEX_ALARM_TYPE = 0
```

Comment:
Yeah, I agree. If there is a way to directly import the enums let me know!

I'll add links like that. I like the idea of linking to the current version, to prevent a broken link.